ext/strings_test.go

```diff
@@ -105,6 +105,10 @@ var stringTests = []struct {
 	// Upper ASCII tests.
 	{expr: `'tacoCat'.upperAscii() == 'TACOCAT'`},
 	{expr: `'tacoCαt'.upperAscii() == 'TACOCαT'`},
+	// Reverse tests.
+	{expr: `'gums'.reverse() == 'smug'`},
+	{expr: `'palindromes'.reverse() == 'semordnilap'`},
+	{expr: `'John Smith'.reverse() == 'htimS nhoJ'`},
```

Comment:
Could you please add some unicode characters in the reverse? While it's clear in the implementation that you're swapping runes and not bytes, for someone else who wants to implement this function, such a test would be useful for conformance checks.